osmosis-1/osmosis.zone_assets.json

```diff
@@ -1680,13 +1680,6 @@
       "osmosis_main": false,
       "osmosis_frontier": true
     },
-    {
```

Comment:
why is this deleting DGL?

Reply:
No idea my fork was completely off somehow. I recloned it and opened a brand new cleaner PR